templates/core/sitemap.xml

```diff
@@ -2,7 +2,7 @@
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
     {% for release in releases -%}
         <url>
-            <loc>https://docs.rs/{{ release[0] }}</loc>
+            <loc>https://docs.rs/{{ release[0] | urlencode }}</loc>
```

Comment:
The default behavior is to html encode, this just takes over from that with url encoding instead. It's a difference that probably doesn't matter a whole lot, so either works

Reply:
I think we actually do want HTML encoding here - the point of escaping is to avoid issues like a url of `"><script>...` which means we need to escape for the current context, not for a url.